### PR TITLE
chore: Download submodules in the CI checkouts

### PR DESCRIPTION
**Pull request by spencergilbert**

### CODE REVIEW

File: .github/audit.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ jobs:
   security_audit:
     runs-on: ubuntu-20.04
     steps:
-      - uses: actions/checkout@v1
+      - uses: actions/checkout@v3
```
Comment (spencergilbert) [inline .github/audit.yml:14]: Unrelated but surprising this wasn't bumped by dependabot

File: .github/workflows/regression.yml
Comment (spencergilbert) [inline .github/workflows/regression.yml:1]: Checked with Kyle, the two removed checkouts were extraneous.

**Review by neuronull — APPROVED** [review]

File: .github/workflows/regression.yml
Comment (spencergilbert) [reply .github/workflows/regression.yml:1]: APPARENTLY NOT

**Pull request merged**